### DISCUSSION TITLE
Syncing Obsidian with Git

### DISCUSSION THREAD

**Discussion by mogigi** [isontheline/pro.webssh.net#743] [Q&A]
Hi, can webssh on ipad be used to sync my obsidian vaults with git? I can't figure out how to get the obsidian-git plugin to work on iPad using ssh to my remote git repo. After looking through tons of tutorials, I was able to get syncing via desktop working.

My setup:
macos used zsh terminal with git (latest version via brew), github desktop app, obsidian desktop app. I just figured out how to get the obsidian-git plugin on desktop to clone my github repo and actually see my folders/files to be able to edit within the obsidian desktop app. The issue I keep having is getting this synced on my iPad obsidian. I've tried iPad github app and working copy app, but the obsidian-git plugin keeps saying it doesn't recognize ssh transport protocol. I have ssh keys configured in github.com account and on my desktop. I'm using only the public keys to connect between github.com and apps. What do I need to do on iPad in order to have the obsidian-git plugin to push/pull/backup etc using my ssh git URL?

thanks

**Comment by isontheline** [comment]
Hi @mogigi 👋

I'm sorry I don't understand your need 😬

Does anyone can help @mogigi with it's request about obsidian-git?

Thank you
